File: Basics_with_JavaScript/56 Conditional Statements/More Exersises/06. Flower Shop.js
function vladi(input){

    let magnolia = 3.25, zumbul = 4, rose = 3.5, cactus = 8;
    let magnoliaQuantity = Number(input[0])
    let zumbulQuantity = Number(input[1])
    let roseQuantity = Number(input[2])
    let cactusQuantity = Number(input[3])
    let presentPrice = Number(input[4])
    let total = magnolia * magnoliaQuantity + zumbul * zumbulQuantity + rose * roseQuantity + cactus * cactusQuantity
    total = 0.95 * total
    let difference = Math.abs(total - presentPrice)
    if (total >= presentPrice) {
        difference = Math.floor(difference)
        console.log(`She is left with ${difference} leva.`);
    } else {
        difference = Math.ceil(difference)
        console.log(`She will have to borrow ${difference} leva.`);
    }

    }
   
    vladi([2,3,5,1,50]);
    vladi([15,7,5,10,100]);

File: Basics_with_Python/PythonBasicCourse/2_Conditional_Statements/6_Conditional_Statements_Excercises_Exercise/more_exercises/06._Flower_Shop.py
import math
magnolia = int(input())
zumbul = int(input())
roses = int(input())
cactus = int(input())
gift_amount = float(input())
total = (magnolia * 3.25 + zumbul * 4 + roses * 3.50 + cactus * 8) * 0.95
difference = abs(total - gift_amount)
if total >= gift_amount:
    print(f"She is left with {math.floor(difference)} leva.")
else:
    print(f"She will have to borrow {math.ceil(difference)} leva.")